src/MetaMorpho.sol

```diff
@@ -464,6 +470,13 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         emit EventsLib.TransferRewards(_msgSender(), rewardsRecipient, token, amount);
     }
 
+    /* MORPHO GETTERS */
+
+    /// @notice Returns the expected supply asset balance of the vault on the market defined by `marketParams`.
+    function expectedSupplyAssets(MarketParams memory marketParams) external view returns (uint256) {
+        return MORPHO.expectedSupplyBalance(marketParams, address(this));
```

Comment:
Maybe we should point to Morpho Blue branch with the renaming.

In this case we should rename `_supplyBalance` as well

Reply:
[Related discussion](https://morpholabs.slack.com/archives/C02N7CSP91C/p1699019069613909) because there can have some implications if we to add `get` before getters